Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/AdapterBuilder.cpp

```diff
@@ -21,6 +21,23 @@ namespace AZ::DocumentPropertyEditor
         CurrentNode()[attribute] = AZStd::move(value);
     }
 
+    void AdapterBuilder::OnEditorChanged(AZStd::function<void(const Dom::Path&, const Dom::Value&)> onChangedCallback)
+    {
+        // The value path is the first child of the PropertyEditor node
+        Dom::Path changedPath = GetCurrentPath() / 0;
```

Comment:
What does dividing by 0 mean in this context?

Reply:
Thanks! I just wanted to know what it was doing, is Dom::Path different than AZ::IO::Path?

Reply:
It's slightly different (though the API is similar) - it's for addressing space in a Document Object Model similar to [JSON Pointer](https://datatracker.ietf.org/doc/html/rfc6901), so our implementation is a vector of `Name`s (for object keys) and `size_t`s (for child indices).